lib/ansible/modules/remote_management/manageiq/manageiq_policies.py

```diff
@@ -0,0 +1,320 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# (c) 2017, Daniel Korn <korndaniel1@gmail.com>
+# (c) 2017, Yaacov Zamir <yzamir@redhat.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+
+DOCUMENTATION = '''
+
+module: manageiq_policies
+
+short_description: Management of resource policy_profiles in ManageIQ.
+extends_documentation_fragment: manageiq
+version_added: '2.5'
+author: Daniel Korn (@dkorn)
+description:
+  - The manageiq_policies module supports adding and deleting policy_profiles in ManageIQ.
+
+options:
+  state:
+    description:
+      - absent - policy_profiles should not exist,
+      - present - policy_profiles should exist,
+      - list - list current policy_profiles and policies.
+    required: False
+    choices: ['absent', 'present', 'list']
+    default: 'present'
+  policy_profiles:
+    description:
+      - list of dictionaries, each includes the policy_profile 'name' key.
+      - required if state is present or absent.
+    required: false
+    default: null
+  resource_type:
+    description:
+      - the type of the resource to which the profile should be [un]assigned
+    required: true
+    choices: ['provider', 'host', 'vm', 'blueprint', 'category', 'cluster',
+        'data store', 'group', 'resource pool', 'service', 'service template',
+        'template', 'tenant', 'user']
+    default: null
+  resource_name:
+    description:
+      - the name of the resource to which the profile should be [un]assigned
+    required: true
+    default: null
+'''
+
+EXAMPLES = '''
+- name: Assign new policy_profile for a provider in ManageIQ
+  manageiq_policies:
+    resource_name: 'EngLab'
+    resource_type: 'provider'
+    policy_profiles:
+      - name: openscap profile
+    manageiq_connection:
+      url: 'http://127.0.0.1:3000'
+      username: 'admin'
+      password: 'smartvm'
+      verify_ssl: False
+
+- name: Unassign a policy_profile for a provider in ManageIQ
+  manageiq_policies:
+    state: absent
+    resource_name: 'EngLab'
+    resource_type: 'provider'
+    policy_profiles:
+      - name: openscap profile
+    manageiq_connection:
+      url: 'http://127.0.0.1:3000'
+      username: 'admin'
+      password: 'smartvm'
+      verify_ssl: False
+
+- name: List current policy_profile and policies for a provider in ManageIQ
+  manageiq_policies:
+    state: list
+    resource_name: 'EngLab'
+    resource_type: 'provider'
+    manageiq_connection:
+      url: 'http://127.0.0.1:3000'
+      username: 'admin'
+      password: 'smartvm'
+      verify_ssl: False
+'''
+
+RETURN = '''
```

Comment:
not a blocker but I am missing some docs here, there are a few returns which gets returned under various conditions.

Reply:
👍

Reply:
@resmo added a return example for the `list` option

Reply:
hmm the key should correspond to the key you return on line https://github.com/ansible/ansible/pull/32892/files#diff-2bcb1f907ba628a71186846aee8852bdR342 IMHO 'profiles' 

IMHO or did I miss anything?
~~~yaml
profiles:
    description:
      - List current policy_profile and policies for a provider in ManageIQ
    returned: state=list
    type: list
    sample: '[
                {
                    "policies": [
                        {
                            "active": true,
                            "description": "OpenSCAP",
                            "name": "openscap policy"
                        },
                        {
                            "active": true,
                            "description": "Analyse incoming container images",
                            "name": "analyse incoming container images"
                        },
                        {
                            "active": true,
                            "description": "Schedule compliance after smart state analysis",
                            "name": "schedule compliance after smart state analysis"
                        }
                    ],
                    "profile_description": "OpenSCAP profile",
                    "profile_name": "openscap profile"
                }
            ]'
~~~

Reply:
> IMHO or did I miss anything?

It was fixed with a rebase, so you did miss it :-(, look here:
https://github.com/ansible/ansible/pull/32892/files#diff-2bcb1f907ba628a71186846aee8852bdR105

Reply:
I am merging and creating a PR for you to review. :)